### PR TITLE
Updated paas content to the new expressions and string constants syntax

### PR DESCRIPTION
**Pull request by mihaitusa**
Align with our latest syntax conventions.

### CODE REVIEW

File: content/io/cloudslang/base/comparisons/equals.sl [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+operation:
+  name: equals
+  inputs:
+    - first
+    - second
+  action:
+    python_script: |
+      eq = first == second
+  results:
+    - EQUALS: eq
```
Comment (Bonczidai) [inline content/io/cloudslang/base/comparisons/equals.sl]: should be expression

File: content/io/cloudslang/base/json/add_value.sl [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        else:
+          decoded = value
+        encoded = json.dumps(decoded)
+        return_code = '0'
+        return_result = 'Parsing successful.'
+      except Exception as ex:
+        return_result = ex
+        return_code = '-1'
+  outputs:
+    - json_output: encoded if return_code == '0' else ''
```
Comment (Bonczidai) [inline content/io/cloudslang/base/json/add_value.sl]: should be expression

File: content/io/cloudslang/base/json/add_value.sl [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    - return_result
+    - return_code
+    - error_message: return_result if return_code == '-1' else ''
```
Comment (Bonczidai) [inline content/io/cloudslang/base/json/add_value.sl]: should be expression

File: content/io/cloudslang/base/json/add_value.sl [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  results:
+    - SUCCESS: return_code == '0'
```
Comment (Bonczidai) [inline content/io/cloudslang/base/json/add_value.sl]: should be expression

File: content/io/cloudslang/base/json/equals.sl [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+        decoded2 = json.loads(json_input2)
+        return_code = '0'
+        return_result = 'Parsing successful.'
+      except Exception as ex:
+        return_result = ex
+        return_code = '-1'
+  outputs:
+    - return_result
+    - return_code
+    - error_message: return_result if return_code == '-1' else ''
```
Comment (Bonczidai) [inline content/io/cloudslang/base/json/equals.sl]: should be expression

File: content/io/cloudslang/base/json/equals.sl [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+  results:
+    - EQUALS: return_code == '0' and decoded1 == decoded2
```
Comment (Bonczidai) [inline content/io/cloudslang/base/json/equals.sl]: should be expression

File: content/io/cloudslang/base/json/equals.sl [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+    - NOT_EQUALS: return_code == '0'
```
Comment (Bonczidai) [inline content/io/cloudslang/base/json/equals.sl]: same, should be expression

File: content/io/cloudslang/base/json/get_keys.sl [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+        for key in json_path:
+          decoded = decoded[key]
+        decoded = decoded.keys()
+        return_result = 'Parsing successful.'
+        return_code = '0'
+      except Exception as ex:
+        return_result = ex
+        return_code = '-1'
+  outputs:
+    - keys: decoded if return_code == '0' else ''
```
Comment (Bonczidai) [inline content/io/cloudslang/base/json/get_keys.sl]: should be expression

File: content/io/cloudslang/base/json/get_keys.sl [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+    - return_result
+    - return_code
+    - error_message: return_result if return_code == '-1' else ''
```
Comment (Bonczidai) [inline content/io/cloudslang/base/json/get_keys.sl]: should be expression

File: content/io/cloudslang/base/json/get_keys.sl [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+  results:
+    - SUCCESS: return_code == '0'
```
Comment (Bonczidai) [inline content/io/cloudslang/base/json/get_keys.sl]: should be expression

File: content/io/cloudslang/base/json/get_value.sl [hunk 11]
```diff
@@ -24,16 +25,16 @@
...
   action:
     python_script: |
       try:
         import json
         decoded = json.loads(json_input)
-        for key in key_list:
+        for key in json_path:
           decoded = decoded[key]
         return_code = '0'
         return_result = 'Parsing successful.'
```
Comment (Bonczidai) [inline content/io/cloudslang/base/json/get_value.sl]: output & results here should be expressions

File: content/io/cloudslang/cloud_provider/digital_ocean/v2/droplets/create_droplet.sl [hunk 12]
```diff
@@ -118,9 +118,9 @@ flow:
...
     - extract_droplet_id:
         do:
-          json.get_value_from_json:
+          json.get_value:
             - json_input: response
-            - key_list: ["'droplet'", "'id'"]
+            - json_path: ["'droplet'", "'id'"]
         publish:
           - droplet_id: value
   outputs:
```
Comment (Bonczidai) [inline content/io/cloudslang/cloud_provider/digital_ocean/v2/droplets/create_droplet.sl]: output & results here should be expressions

File: content/io/cloudslang/paas/openshift/applications/create_application.sl [hunk 13]
```diff
@@ -82,21 +83,21 @@ flow:
...
-            - content_type: "'application/json'"
-            - application_name_string: "'\"name\":\"' + application_name + '\",'"
-            - cartridge_string: "'\"cartridge\":[' + cartridge_str + ']'"
-            - scale_string: "',\"scale\":' + str(scale).lower()"
-            - gear_profile_string: "',\"gear_profile\":\"' + gear_profile + '\"' if gear_profile else ''"
-            - initial_git_url_string: "',\"initial_git_url\":\"' + initial_git_url + '\"' if initial_git_url else ''"
-            - body: "'{' + application_name_string + cartridge_string + gear_profile_string + initial_git_url_string + scale_string + '}'"
-            - headers: "'Accept: application/json'"
+            - content_type: 'application/json'
+            - application_name_string: "${'\"name\":\"' + application_name + '\",'}"
```
Comment (Bonczidai) [inline content/io/cloudslang/paas/openshift/applications/create_application.sl]: please check trhat you really need `"` to enclose `${...}` in these cases

File: content/io/cloudslang/paas/openshift/applications/delete_application.sl [hunk 14]
```diff
@@ -54,15 +54,15 @@ flow:
     - delete_app:
         do:
           rest.http_client_delete:
-            - url: "'https://' + host + '/broker/rest/domains/' + domain + '/applications/' + application_name"
+            - url: "${'https://' + host + '/broker/rest/domains/' + domain + '/applications/' + application_name}"
```
Comment (Bonczidai) [inline content/io/cloudslang/paas/openshift/applications/delete_application.sl]: please check trhat you really need `"` to enclose `${...}` in these cases

File: content/io/cloudslang/paas/openshift/applications/list_applications.sl [hunk 15]
```diff
@@ -52,15 +52,15 @@ flow:
     - list_apps:
         do:
           rest.http_client_get:
-            - url: "'https://' + host + '/broker/rest/domains/' + domain + '/applications'"
+            - url: "${'https://' + host + '/broker/rest/domains/' + domain + '/applications'}"
```
Comment (Bonczidai) [inline content/io/cloudslang/paas/openshift/applications/list_applications.sl]: please check trhat you really need `"` to enclose `${...}` in these cases

File: content/io/cloudslang/paas/openshift/applications/restart_application.sl [hunk 16]
```diff
@@ -54,16 +54,16 @@ flow:
     - restart_app:
         do:
           rest.http_client_post:
-            - url: "'https://' + host + '/broker/rest/domains/' + domain + '/applications/' + application_name + '/events'"
+            - url: "${'https://' + host + '/broker/rest/domains/' + domain + '/applications/' + application_name + '/events'}"
```
Comment (Bonczidai) [inline content/io/cloudslang/paas/openshift/applications/restart_application.sl]: please check that you really need `"` to enclose `${...}` in these cases

File: content/io/cloudslang/paas/openshift/applications/scale_down_application.sl [hunk 17]
```diff
@@ -54,16 +54,16 @@ flow:
     - scale_down_app:
         do:
           rest.http_client_post:
-            - url: "'https://' + host + '/broker/rest/domains/' + domain + '/applications/' + application_name + '/events'"
+            - url: "${'https://' + host + '/broker/rest/domains/' + domain + '/applications/' + application_name + '/events'}"
```
Comment (Bonczidai) [inline content/io/cloudslang/paas/openshift/applications/scale_down_application.sl]: please check that you really need `"` to enclose `${...}` in these cases -- please check this arg for each flow (not mentioning again) :)

File: content/io/cloudslang/paas/openshift/cartridges/add_cartridge.sl [hunk 18]
```diff
@@ -65,8 +66,8 @@ flow:
             - proxy_username
             - proxy_password
             - content_type: "'application/json'"
```
Comment (Bonczidai) [inline content/io/cloudslang/paas/openshift/cartridges/add_cartridge.sl]: on pair of quotes

File: content/io/cloudslang/paas/openshift/cartridges/add_cartridge.sl [hunk 19]
```diff
@@ -65,8 +66,8 @@ flow:
...
-            - body: "'{\"cartridge\":\"' + cartridge + '\"}'"
-            - headers: "'Accept: application/json'"
+            - body: "${'{\"cartridge\":\"' + cartridge + '\"}'}"
```
Comment (Bonczidai) [inline content/io/cloudslang/paas/openshift/cartridges/add_cartridge.sl]: please check that you really need " to enclose ${...} in these cases -- please check this arg for each flow (not mentioning again) :)

File: content/io/cloudslang/paas/stackato/applications/create_application.sl [hunk 20]
```diff
@@ -79,16 +80,16 @@ flow:
...
             - password
             - proxy_host
             - proxy_port
             - proxy_username
             - proxy_password
-            - body: "'{\"name\":\"' + application_name + '\",\"space_guid\":\"' + space_guid + '\",\"memory\":1024,\"instances\":1}'"
-            - headers: "'Authorization: bearer ' + token"
-            - content_type: "'application/json'"
+            - body: "${'{\"name\":\"' + application_name + '\",\"space_guid\":\"' + space_guid + '\",\"memory\":1024,\"instances\":1}'}"
+            - headers: "${'Authorization: bearer ' + token}"
```
Comment (Bonczidai) [inline content/io/cloudslang/paas/stackato/applications/create_application.sl]: please check that you really need " to enclose ${...} in these cases -- please check this arg for each flow (not mentioning again) :)

**Comment by Bonczidai** [comment]
:+1: please handle small comments

File: test/io/cloudslang/paas/openshift/applications/test_create_application.sl [hunk 21]
```diff
@@ -73,8 +73,8 @@ flow:
     - check_result:
         do:
           lists.compare_lists:
-            - list_1: [str(error_message), int(return_code), int(status_code)]
-            - list_2: ["''", 0, 201]
+            - list_1: ${[str(error_message), int(return_code), int(status_code)]}
+            - list_2: ${['', 0, 201]}
```
Comment (Bonczidai) [inline test/io/cloudslang/paas/openshift/applications/test_create_application.sl]: if a list (`[...]`) does not contain an expression (only primitive values) do not use expression syntax - please check all over - search after `${[`

**Comment by mihaitusa** [comment]
@Bonczidai Please review it once again!

**Pull request merged**